db/views/analytics_clients_v01.sql

```diff
@@ -0,0 +1 @@
+SELECT * FROM "Client" WHERE "DateDeleted" is NULL
```

Comment:
Should we be removing app-internal fields from these models? Maybe just export the CamelCase fields? 
```sql
-- Hmis::Hud::Client.column_names.grep(/\A[A-Z]/ ).sort.map { |f| "\"#{f}\""}.join
(",\n")
SELECT id, "AdditionalRaceEthnicity",
"AfghanistanOEF",
- ...
"WorldWarII",
"YearEnteredService",
"YearSeparated" FROM "Client" WHERE "DateDeleted" is NULL
```


Maybe even fix the casing in our fields?
```sql
-- `Hmis::Hud::Client.column_names.grep(/\A[A-Z]/ ).sort.map { |f| "\"#{f}\" as #{f.underscore}"}.join(",")`
SELECT id,  
"AdditionalRaceEthnicity" as additional_race_ethnicity,
"AfghanistanOEF" as afghanistan_oef,
-- ...
"WorldWarII" as world_war_ii,
"YearEnteredService" as year_entered_service,
"YearSeparated" as year_separated
 FROM "Client" WHERE "DateDeleted" is NULL
```

Reply:
I want to discuss, probably with @Jacobihi, if we should limit the columns to those in the HMIS CSVs (essentially.)  At the moment we are using the PG copier to pull the whole table and then do the refactoring in DBT (or at least that's my understanding.)

Reply:
That is what we're doing currently. I think handling all the transformation in one place is preferable so I think renaming the columns downstream is fine, because at this juncture it would be more work to re-map newly named fields (because we have a pattern of decoding the hud fields and call the column `CamelCase` -> `camel_case_code`

If there's data as shouldn't be exposed then I certainly agree we don't need to copy it, so as long as all the internal ids that make the joins work are present the HMIS csvs can strip to only the CamelCase fields + id.

Reply:
Chatted with @Jacobihi and we're going to leave these as is for V1.  This replicates the same data footprint we currently have but moves it to the analytics schema, which should mean we can pivot to a simpler mechanism to ingest it.  Once that work has been completed we can revisit if we want to decrease the available footprint by adjusting the views.